Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+WORKDIR /usr/src/app
```

Comment:
Having the work dir in the engine's source dir is unusual for us. Should we move the work dir to `/code` later on in this file?